test/example-config.yml

```diff
@@ -8,6 +8,32 @@ subarrays:
     namespace: sdparray1
   subarray2:
     namespace: sdparray2
+antennas:
+  - "m000"
+  - "m001"
+  - "m002"
+  - "m003"
 components:
   ingest:
-    image: cchristelis/ingest:0.5
+    default_image: "harbor.sdp.kat.ac.za:443/infra/pocingest:0.5"
+    resources:
+      - "allocation_info":
+          "role": "realtime"
+        "name": "cpus"
+        "role": "realtime"
+        "scalar":
+          "value": 1.2
+        "type": "SCALAR"
+  telstate:
+    docker_image: "harbor.sdp.kat.ac.za:443/infra/pocbatch_setup:0.4"
+  calibration:
+    docker_image: "harbor.sdp.kat.ac.za:443/infra/poccalibrator:0.1"
+    resources: {
```

Comment:
I cannot make up my mind about this. Having the resource definition in Argo format makes this very exact and power full but a bit unnecessarily long-winded for a config. I am leaning towards this format (Argo) used here.
Maybe allow for requests to be left out and if it is not defined then define requests based on limits in the code. Then at least we remove the duplication in the config.

Reply:
Done. It looks a lot cleaner now :)